autoscale-api/src/main/java/com/sequenceiq/periscope/api/model/AutoscaleClusterState.java

```diff
@@ -11,6 +12,9 @@
     @ApiModelProperty(ClusterAutoscaleState.ENABLE_AUTOSCALING)
     private boolean enableAutoscaling;
 
+    @ApiModelProperty(ClusterStopStartScalingState.ENABLE_STOP_START_SCALING)
```

Comment:
How are defaults being handled here? Specifically, if enableAutoscaleForClusterName (DistroXAutoScaleClusterV1Endpoint) gets invoked with either enableAutoScale set or useStopStartScaling set (not both) - will this end up resetting the value of the other parameter?

The API should allow setting these values independently. If I'm not mistaken, calling it with a single parameter will end up using the default value for the other one - which essentially resets existing state.

Reply:
Yes, upon providing only one parameter, for ex: `enableAutoscaling` as true, the other parameter, `useStopStartMechanism` is set to its default value: false, which ends up unintentionally changing the existing state. Updated the implementation so that when only one of these params is provided, the other one isn't altered, i.e, the values can be toggled independently.